Volume/Views/DebriefArticleView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  DebriefArticleView.swift
+//  Volume
+//
+//  Created by Amy Chin Siu Huang on 12/2/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import AppDevAnalytics
+import Combine
+import SwiftUI
+import SDWebImageSwiftUI
+
+struct DebriefArticleView: View {
+    @State private var cancellableShoutoutMutation: AnyCancellable?
+    @EnvironmentObject private var userData: UserData
+    let header: String
+    let article: Article
+
+    var body: some View {
```

Comment:
This is massive. Break it up into sections like we do in [`PublicationList`](https://github.com/cuappdev/volume-ios/blob/master/Volume/Views/MainView/PublicationList.swift)